centos/deprecated.md

```diff
@@ -1 +1 @@
-This image is no longer supported/maintained (non-EOL tags last updated November 16, 2020, [docker-library/official-images#9102](https://github.com/docker-library/official-images/pull/9102); see also https://www.centos.org/centos-linux-eol/ and [docker-library/docs#2205](https://github.com/docker-library/docs/pull/2205)). Please adjust your usage accordingly.
+*All* tags of this image are EOL ([June 30, 2024](https://www.redhat.com/en/topics/linux/centos-linux-eol) / [docker-library/official-images#17094](https://github.com/docker-library/official-images/pull/17094), but it was inactive/unmaintained long before that: meaningful tags were last updated November 16, 2020, [docker-library/official-images#9102](https://github.com/docker-library/official-images/pull/9102); see also https://www.centos.org/centos-linux-eol/ and [docker-library/docs#2205](https://github.com/docker-library/docs/pull/2205)). Please adjust your usage accordingly.
```

Comment:
```suggestion
*All* tags of this image are EOL ([June 30, 2024](https://www.redhat.com/en/topics/linux/centos-linux-eol) / [docker-library/official-images#17094](https://github.com/docker-library/official-images/pull/17094), although the last meaningful update was November 16, 2020, long before the EOL date [docker-library/official-images#9102](https://github.com/docker-library/official-images/pull/9102); see also https://www.centos.org/centos-linux-eol/ and [docker-library/docs#2205](https://github.com/docker-library/docs/pull/2205)). Please adjust your usage accordingly.
```

Reply:
Applied with the minor addition of a colon after "long before the EOL date":

> *All* tags of this image are EOL ([June 30, 2024](https://www.redhat.com/en/topics/linux/centos-linux-eol) / [docker-library/official-images#17094](https://github.com/docker-library/official-images/pull/17094), although the last meaningful update was November 16, 2020, long before the EOL date: [docker-library/official-images#9102](https://github.com/docker-library/official-images/pull/9102); see also https://www.centos.org/centos-linux-eol/ and [docker-library/docs#2205](https://github.com/docker-library/docs/pull/2205)). Please adjust your usage accordingly.

vs

> *All* tags of this image are EOL ([June 30, 2024](https://www.redhat.com/en/topics/linux/centos-linux-eol) / [docker-library/official-images#17094](https://github.com/docker-library/official-images/pull/17094), although the last meaningful update was November 16, 2020, long before the EOL date [docker-library/official-images#9102](https://github.com/docker-library/official-images/pull/9102); see also https://www.centos.org/centos-linux-eol/ and [docker-library/docs#2205](https://github.com/docker-library/docs/pull/2205)). Please adjust your usage accordingly.